### PR TITLE
Code comments cleanup

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Hey @alexagranov actually I decided to send this mini PR to clean up the `XXX` codetag comments from previously merged files (and other minor updates).

### CODE REVIEW

File: test/Controllers/SearchTest.php [hunk 1]
```diff
@@ -38,7 +36,7 @@ public function setUp() {
      * GonebusyLib\Controllers\SearchController::searchQuery()
      */
     public function testSearchQuery() {
-        $query = "testing@gonebusy.com"; // XXX Is this working?
```
Comment (jorgeorpinel) [inline test/Controllers/SearchTest.php]: The only thing to notice though, is that this one comment I'm removing may indicate a bug in the search. I was guessing searching for that email address would return the Sandbox user but it doesn't. In fact I don't think search is returning users in general. May want to check that (:

File: test/Controllers/ServicesTest.php [hunk 2]
```diff
@@ -44,11 +43,10 @@ private function serviceBody($type) {
             case 'create':
                 return new CreateServiceBody(
                     "description", // REQUIRED
-                    15, // duration REQUIRED XXX 0 not accepted but NULL turns into 0
-                    // XXX No max_duration in SDK
```
Comment (jorgeorpinel) [inline test/Controllers/ServicesTest.php]: Wait, there's more! This comment also signaled another posible bug. Seem the SDK is not up to date with API? `max_duration` isn't being returned. FYI. Thanks

File: test/Controllers/SearchTest.php (hunk 1 above)
Comment (alexagranov) [inline test/Controllers/SearchTest.php]: No that's correct for now.  Search is quite limited right now.

File: test/Controllers/ServicesTest.php (hunk 2 above)
Comment (alexagranov) [inline test/Controllers/ServicesTest.php]: See, this is why I :heart: specs 🤓 

Somehow `:max_duration` was dropped in a prior commit on a couple of the request bodies.  I've fixed in v0.0.9 #21 

Please pull latest version and retry.

**Comment by jorgeorpinel** [comment]
Updated per 0.0.9 @alexagranov . Merge this one first is posible. Thanks

**Pull request merged**